app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, 
+                distance_from_city_center: float, 
+                clean_power: int,
+                average_rating: float,
+                count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+        def serve_cars(cars: list) -> int:
+            total_income = 0
+            for car in cars:
+                if type(car.clean_mark) == "int" and type(self.clean_power) == "int":
+                    if car.clean_mark < self.clean_power:
+                        income = car.comfort_class * 1.25
+                        total_income += income
+                        car.clean_mark = self.clean_power
+                else:
+                    print("Incorrect data type")
+            return round(total_income, 1)
+
+        def calculate_washing_price(self, car: Car) -> float:
+            total_price = car.comfort_class * (self.clean_power - car.clean_mark) * self.average_rating / self.distance_from_city_center
+            return round(total_price, 1)
+
+        def wash_single_car(car: Car) -> None:
+            if type(car.clean_mark) == "int" and type(self.clean_power) == "int":
```

Comment:
The type check for `car.clean_mark` and `self.clean_power` should use `isinstance()` instead of `type() == "int"`. This is a more Pythonic way to check types.